productivity-suite/app/legacy-app/index.html

```diff
@@ -178,6 +178,13 @@
         max-width: min(calc(55rem + 2rem), calc(90vw));
       }
 
+    :not(piercing-fragment-outlet) > piercing-fragment-host:last-of-type::after {
+      content: '';
+      position: absolute;
```

Comment:
yes, but we aren't actually expanding the container but adding something below it (so that it doesn't touches the bottom of the screen)

for example:

![Screenshot 2022-11-15 at 18 38 35](https://user-images.githubusercontent.com/61631103/201999590-3d718dbf-ac77-4759-a66a-a75ef2fc4d48.png)

![Screenshot 2022-11-15 at 18 38 43](https://user-images.githubusercontent.com/61631103/201999610-64cbac85-919f-4899-a914-455520c3026c.png)

as you can see we're adding an extra bit there outside the container so that it "pushes down" the bottom of the screen

this works but it is not very elegant... if you'd got an alternative idea in mind that's be awesome 😃